CodegenCLI/README.md

```diff
@@ -0,0 +1,60 @@
+# CodegenCLI
+
+```
+OVERVIEW: A command line utility for Apollo iOS code generation.
+
+USAGE: apollo-ios-codegen <subcommand>
+
+OPTIONS:
+  --version               Show the version.
+  -h, --help              Show help information.
+
+SUBCOMMANDS:
+  init                    Initialize a new configuration with defaults.
+  validate                Validate a configuration file or JSON formatted string.
+  generate                Generate Swift source code based on a code generation configuration.
+
+  See 'apollo-ios-codegen help <subcommand>' for detailed help.
+```
+
+## Initialize
+```
+OVERVIEW: Initialize a new configuration with defaults.
+
+USAGE: apollo-ios-codegen init [--output <output>] [--path <path>] [--overwrite]
+
+OPTIONS:
+  -o, --output <output>   Destination for the new configuration. (default: file)
+  -p, --path <path>       Write the configuration to a file at the path. (default: ./apollo-codegen-config.json)
+  -w, --overwrite         Overwrite any file at --path.
```

Comment:
Can we add "If init is called without --overwrite and a config file already exists at the output path, the command will fail."